### ISSUE TITLE
Arity exception with leiningen 2.7.0

### ISSUE THREAD

**Issue by mtnygard** [sunng87/lein-bootclasspath-deps#1]
Not sure precisely when this arose, but in leiningen 2.7.0:

```
clojure.lang.ArityException: Wrong number of args (2) passed to: classpath/get-dependencies
 at clojure.lang.AFn.throwArity (AFn.java:429)
    clojure.lang.RestFn.invoke (RestFn.java:427)
    lein_bootclasspath_deps.plugin$middleware.invokeStatic (plugin.clj:12)
    lein_bootclasspath_deps.plugin$middleware.invoke (plugin.clj:6)
    clojure.lang.Var.invoke (Var.java:379)
    leiningen.core.project$apply_middleware.invokeStatic (project.clj:750)
    leiningen.core.project$apply_middleware.invoke (project.clj:743)
    clojure.core.protocols$fn__6755.invokeStatic (protocols.clj:167)
    clojure.core.protocols/fn (protocols.clj:124)
    clojure.core.protocols$fn__6710$G__6705__6719.invoke (protocols.clj:19)
    clojure.core.protocols$seq_reduce.invokeStatic (protocols.clj:31)
    clojure.core.protocols$fn__6738.invokeStatic (protocols.clj:75)
    clojure.core.protocols/fn (protocols.clj:75)
    clojure.core.protocols$fn__6684$G__6679__6697.invoke (protocols.clj:13)
    clojure.core$reduce.invokeStatic (core.clj:6545)
    clojure.core$reduce.invoke (core.clj:6527)
    leiningen.core.project$apply_middleware.invokeStatic (project.clj:745)
    leiningen.core.project$apply_middleware.invoke (project.clj:743)
    leiningen.core.project$activate_middleware.invokeStatic (project.clj:777)
    leiningen.core.project$activate_middleware.invoke (project.clj:773)
    leiningen.core.project$init_project.invokeStatic (project.clj:902)
    leiningen.core.project$init_project.invoke (project.clj:892)
    leiningen.core.project$read.invokeStatic (project.clj:973)
    leiningen.core.project$read.invoke (project.clj:970)
    leiningen.core.project$read.invokeStatic (project.clj:974)
    leiningen.core.project$read.invoke (project.clj:970)
    leiningen.core.main$_main$fn__6165.invoke (main.clj:397)
    leiningen.core.main$_main.invokeStatic (main.clj:394)
    leiningen.core.main$_main.doInvoke (main.clj:391)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.lang.Var.invoke (Var.java:379)
    clojure.lang.AFn.applyToHelper (AFn.java:154)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invokeStatic (core.clj:646)
    clojure.main$main_opt.invokeStatic (main.clj:314)
    clojure.main$main_opt.invoke (main.clj:310)
    clojure.main$main.invokeStatic (main.clj:421)
    clojure.main$main.doInvoke (main.clj:384)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)
```

**Comment by mtnygard** [comment]
On further investigation, it appears to have been introduced with Leiningen's support for "managed dependencies" in https://github.com/technomancy/leiningen/issues/2067

**Comment by sunng87** [comment]
Thank you for reporting! A fix is released to clojars as 0.3.0.
